src/frontend/src/lib/types/dappDescription.ts

```diff
@@ -14,13 +14,16 @@ const dAppDescriptionSchema = z.object({
 	description: z.string(),
 	stats: z.string(),
 	logo: z.string(),
+	callToAction: z.string().optional(),
```

Comment:
given that `DappDescription` matches the types of the portal and II, can you rather add the new specific fields for Oisy as extension?

For example:

interface DappDescription {}
interface OisyDappDescription extends DappDescription {}

(just used a dummy name to explain what I meant)

Reply:
Yes I like this idea.